### PR TITLE
add missing parens and give vars better names

### PR DESCRIPTION
**Pull request by Ch4s3**

### CODE REVIEW

File: lib/classifier-reborn/lsi.rb [hunk 1]
```diff
@@ -78,7 +78,7 @@ def add_item(item, *categories, &block)
     # will be duck typed via to_s .
     #
     def <<(item)
-      add_item item
+      add_item(item)
```
Comment (parkr) [inline lib/classifier-reborn/lsi.rb]: perhaps we could use `alias_method` here?

File: lib/classifier-reborn/lsi.rb [hunk 2]
```diff
@@ -280,9 +280,9 @@ def scored_categories(doc, cutoff = 0.30, &block)
     # it's supposed to.
     def highest_ranked_stems(doc, count = 3)
       raise 'Requested stem ranking on non-indexed content!' unless @items[doc]
-      arr = node_for_content(doc).lsi_vector.to_a
-      top_n = arr.sort.reverse[0..count - 1]
-      top_n.collect { |x| @word_list.word_for_index(arr.index(x)) }
+      content_vecotr_array = node_for_content(doc).lsi_vector.to_a
```
Comment (parkr) [inline lib/classifier-reborn/lsi.rb]: `vector`? looks like a misspelling

**Comment by parkr** [comment]
:+1:

File: lib/classifier-reborn/lsi.rb (hunk 2 above)
Comment (Ch4s3) [inline lib/classifier-reborn/lsi.rb]: oops.

**Pull request merged**